.github/workflows/build.yml

```diff
@@ -28,6 +28,47 @@ jobs:
         run: make pip
       - name: Build and Test
         run: make build-and-test
```

Comment:
I'm mostly out of ideas here.  

Locally via https://github.com/nektos/act, it works at these commits:
* https://github.com/algorand/pyteal/pull/260/commits/e1ffdadcfaf540686b8d06ea4528ae4cb1461b58
* https://github.com/algorand/pyteal/pull/260/commits/4f269a3a1a63ce6679a0e8187ffb90132b69cb7b
* https://github.com/algorand/pyteal/pull/260/commits/2e8433b0535ae20d2a1004ec564aa244d0e844e9

It's unclear to me how to otherwise debug a failed invocation (e.g. https://github.com/algorand/pyteal/runs/5664812490?check_suite_focus=true).  It's quite possible I'm taking an approach that's a poor fit.

Reply:
Thanks for suggesting `act`. Really useful!!!